docs/source/setup/recipes.rst

```diff
@@ -67,3 +67,222 @@ example, support for rabbitMQ message bus or web support), as well asl supportin
+setting to ``auto`` as shown in the included file:
+
+.. literalinclude:: ../../../examples/deployment/ansible.cfg
+   :caption: ansible.cfg
+   :language: yaml
+   :linenos:
+
+
+Step 1: Prepare configuration files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+VOLTTRON's recipes require there to be two levels of configuration. The ansible inventory
+is used to configure the behavior of the roles and playbooks which constitute the recipe, and the
+platform configurations contain more detailed configuration details of each platform and its
+components. The actual tasks executed consider both sources so that for any particular configuration
+choice there should only need to be a single source of truth. We construct each in the following
+subsections.
+
+The inventory configuration structure available from ansible is quite sophisticated and anything
+described in the `ansible inventory documentation <https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html>`_
+is available for use with recipes. For this example we start with a truely minimal configuration,
+which consists only of the list of our three machines (where we've made the conventional choice
+that the inventory host names will match the VM names in the generated ssh configuration.
+
+.. literalinclude:: ../../../examples/deployment/recipe-inventory.minimal.yml
+   :caption: recipe-inventory.minimal.yml
+   :language: yaml
+   :linenos:
+
+The minimal platform configuration file is a yaml file with a ``config`` key equal to an empty dictionary.
+The default location is a file named the same as the
+inventory host name with extension ``.yml``, in a directory next to the inventory file with name also
+matching the inventory host name. (This location is configurable via inventory variables).
+In this case the instance name is taken to be the inventory host name and all other configs are
+left blank. That configuration looks like:
+
+.. literalinclude:: ../../../examples/deployment/web/web.yml
+   :caption: web/web.yml
+   :language: yaml
+   :linenos:
+
+
+By way of a demonstration, we'll expand the minimal inventory to achieve the following:
+
+* set the volttron_home to be ``~/volttron_home`` on each systems by using a group variable (line 15)
+
+* override the above to be ``~/vhome`` on only the web host by using a host variable (line 6)
+
+* configure collector1 to use the local VOLTTRON source tree rather than downloading from github (line 9)
+
+* install the "drivers" optional feature when bootstrapping the volttron virtualenvironment on collector1 (lines 10-11)
+
+These are all achived with an inventory that looks like:
+
+.. literalinclude:: ../../../examples/deployment/recipe-inventory.yml
+   :caption: recipe-inventory.yml
+   :language: yaml
+   :linenos:
+
+You could expand the inventory to assign values to any of the variables documented in the
+:ref:`recipes-configuration` section. These can be applied to specifc hosts, or to groups per the
```

Comment:
This doesn't show up as a link so not sure what recipes-configuration section nor where to find it.